test/wanda_web/auth/user_detector_test.exs

```diff
@@ -0,0 +1,59 @@
+defmodule WandaWeb.Auth.UserDetectorTest do
+  use WandaWeb.ConnCase, async: false
```

Comment:
Why `async: false`?
Is there a chance to collide with other tests?

Reply:
bad copypasta 🙈